packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -278,10 +279,12 @@ function TokenBalance({
     return <StyledLoader />
   }
 
+  const tokenSymbol = tokenSymbolOverride ?? forToken.symbol
+
   return (
-    <p aria-label={`${forToken.symbol} balance on ${on}`}>
+    <p aria-label={`${tokenSymbol} balance on ${on}`}>
       <span className="font-light">{prefix}</span>
-      <span>
+      <span aria-label={`${tokenSymbol} balance amount on ${on}`}>
```

Comment:
Do we still need the `aria-label` on `p`? It's used only in one place (depositErc20.cy.ts) , maybe we can use the `X balance amount on Y` there too?

Reply:
Good point, yes, removed the `aria-label={${tokenSymbol} balance on ${on}}` code